spec/system/admin/invoice_print_spec.rb

```diff
@@ -300,6 +300,21 @@
             expect(page).to have_content "Total (Excl. tax): $1,458.67"
           end
         end
+
+        context "Line item with variant having variant_unit as 'items'" do
+          before do
+            line_item1.variant.update_columns(variant_unit: "items", display_as: "1 bucket")
```

Comment:
`update_columns` bypasses normal validation. To ensure  more "real-life" conditions we can use:

```suggestion
            line_item1.variant.update!(variant_unit: "items", display_as: "1 bucket")
```